speedy-recovery-client/src/service/FhirDataMappingService.js

```diff
@@ -1,76 +1,71 @@
-// TODO: Many different versions of fields like name, make more resilient (and be careful to work with V3!)
-// TODO: Add different mappings for different FHIR versions (as a proof of concept)
+import {
+  mapAppointmentSTU2,
+  mapConditionSTU2,
+  mapMedicationSTU2,
+  mapPatientToUserSTU2
+} from "./FhirMappingAdapterSTU2";
+import {
+  mapAppointmentSTU3,
+  mapConditionSTU3,
+  mapMedicationSTU3,
+  mapPatientToUserSTU3
+} from "./FhirMappingAdapterSTU3";
 
-// FIXME: Update mapping
-// arrow function with parameter
+const fhirMapPatient = (resource, version) => {
+  if (version) {
```

Comment:
@joshuazeltser Does this solve our problem? What happens when `version` is missing? Method will just implicitly return `undefined`. Is that expected? Have you tried it and actually ran the app with these changes?

Reply:
I don't receive any errors regarding this problem

Reply:
I have added a log if version doesn't exist